change/@ni-nimble-angular-cef992f7-0e96-4b96-8c49-907a8a794bd0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I like a lot of how this is going. My only concern is that taken to the end this pattern results in pretty much nothing in the src folder and a lot of top-level folders. So I think I'm on the boat of keeping all the source inside the top-level `src` folder.

So it'll look like:
```
/src/directives - legacy stuff to make into new entrypoints
/src/table-column - new entrypoint
/src/table - new entrypoint
/src/internal-utilities - new entrypoint
```
But I don't think we need to change anything about the structure inside the `table-column/`, `table/`, and `internal-utilities` folders as you have shown them.

Maybes it's worthwhile to sync on this idea with @atmgrifter00 and @jattasNI to avoid extra churn on the direction?

Reply:
From what I can tell the directory structure you are suggesting isn't supported. The entry point name is automatically determined from the directory hierarchy containing the `ng-package.json` file. Therefore, with the structure you're suggesting, our entry points would be `@ni/nimble-angular/src/table-column`, `@ni/nimble-angular/src/table`, etc. I don't think that's what we want.

I found an issue with a lot of discussion around this topic here: https://github.com/ng-packagr/ng-packagr/issues/900

Reply:
Ah didn't realize that wasn't straightforward. I don't have any deal breaking concerns with the current approach and the public api of it looks good. We can definitely start this way! Thanks for looking into it 👍